PROJECT

```diff
@@ -38,4 +38,17 @@ resources:
   kind: KeystoneEndpoint
   path: github.com/openstack-k8s-operators/keystone-operator/api/v1beta1
   version: v1beta1
```

Comment:
since we now have 'multiple groups' in the same operator should we mark it as multigroup: true? NOTE: this would fundamentally change the structure of the api's directory to accommodate multiple groups as well.